### ISSUE TITLE
bug: whole cody tab is greyed out , nothing visible

### ISSUE THREAD

**Issue by tb-ShashwatDwivedi** [sourcegraph/cody#5308]
Type: <b>Bug</b>

## Extension Information

- Cody Version: 1.32.0
- VS Code Version: 1.90.2
- Extension Host: desktop

##  Steps to Reproduce

1.
2.
3.

## Logs

Extension version: 1.32.0
VS Code version: Code 1.90.2 (5437499feb04f7a586f677b155b039bc2b3669eb, 2024-06-18T22:34:26.404Z)
OS version: Windows_NT x64 10.0.22621
Modes:
Remote OS version: Linux x64 5.4.0-189-generic
Remote OS version: Linux x64 5.15.0-60-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8365U CPU @ 1.60GHz (8 x 1896)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|7.81GB (2.67GB free)|
|Process Argv|--crash-reporter-id 77688af3-8cb1-421b-8e02-7dd17a63265c|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: 192.168.0.19|
|OS|Linux x64 5.4.0-189-generic|
|CPUs|Common KVM processor (6 x 0)|
|Memory (System)|15.44GB (12.16GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: 192.168.0.13|
|OS|Linux x64 5.15.0-60-generic|
|CPUs|Common KVM processor (4 x 0)|
|Memory (System)|3.83GB (2.18GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonnoceb:30805159
asynctok:30898717
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
dsvsc021:30996838
g316j359:31013175
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
dwnewjupyter:31046869
impr_priority:31102340
refactort:31108082
pythonrstrctxt:31112756
wkspc-onlycs-c:31111717
wkspc-ranged-c:31123312
fje88620:31121564

```

</details>

**Comment by ethan2106** [comment]
I have the same problem, it's recurring, I have to **restart the app every 5 minutes.** (paid version)

**Comment by tech-and-ai** [comment]
still does this for me on 1.32.1 which was supposed to resolve this issue.

**Comment by theronburger** [comment]
Is this a dupe of https://github.com/sourcegraph/cody/issues/5296?
I provide possible reasons here. spoiler alert, cody spams state hard and crashes it 
Recover the chat by restarting and going to history. 
Come on SourceGraph, fix this, its a huge pain

**Comment by kalanchan** [comment]
This should be fixed in 1.38, please upgrade and let us know if it's still happening!
